docs/CHANGELOG.md

```diff
@@ -4,8 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-### 2024-02-01
-- Fix: Update `github_actions.yml` file steps versions (https://outsystemsrd.atlassian.net/browse/RMET-2568).
+## [1.0.0]
 
-### 2022-04-12
-Create repository.
+### 2024-02-07
+- Feat: Add workflows to run unit tests and release the library in Maven repo (https://outsystemsrd.atlassian.net/browse/RMET-3975).
+
+### 2025-01-07
+- Feat: Add implementation for whole library, including `getCurrentPosition`, `watchPosition`, and `clearWatch` (https://outsystemsrd.atlassian.net/browse/RMET-3370).
+- Chore: Create repository (https://outsystemsrd.atlassian.net/browse/RMET-3975).
```

Comment:
Since we're going public with the repo, I don't know if we should be pointing to our Jira tickets. 
It's the same thing we've discussed before with the commit messages.

Reply:
You're right, force of habit 😅. I've removed the JIRA ticket links.